instances/measuredSignalType/gammaActivity.jsonld

```diff
@@ -0,0 +1,25 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/measuredSignalType/gammaActivity",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/measuredSignalType",
+  "definition": "Gamma activiation is an electromagnetic oscillation in the frequency range of 25 to 100 Hz (40 Hz is prototypical). [adapted from [wikipedia](https://en.wikipedia.org/wiki/Gamma_wave)]",
```

Comment:
"Gamma activity is a neural oscillation in the high frequency range (typically between 30-150 Hz) arising from synchronous and coherent electrical activity in the brain. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Gamma_wave)]"

@apdavison would you agree with this definition?